priv/templates/migrations/20220415083299_create_collection_events.exs.eex

```diff
@@ -27,11 +27,14 @@ defmodule <%= module_prefix %>.Repo.Migrations.CreateCollectionEvents do
       add :event_type, :event_type, null: false
       add :venue, :text, null: false
       add :slug, :text, null: false
+      add :id, :integer, null: false
       add :token_id, :integer
 
       timestamps()
     end
 
+    create index(:collection_events, [:venue, :slug, :event_type, :id, :token_id, :event_timestamp], unique: true)
```

Comment:
Can this be reduced to `venue+slug+event id`?

Reply:
without `event_timestamp` field timescale wouldn't allow creating hypertable.   Token_id is to be on the safe side.  I'm pretty sure the combo above guarantees uniqueness, but I'm not 100% sure with OpenSea (might have to make it nullable for other venues, so these other attrs are important).    I'm just making sure we're not eliminating anything that came for the first time.     And even with such strict unique index there were plenty of conflicts observed during repeated imports while I was still ironing it out.